modules/shutter_page/cdn_endpoints.tf

```diff
@@ -0,0 +1,18 @@
+resource "azurerm_cdn_endpoint" "shutter_endpoint" {
+  count                     = length(var.shutter_apps)
+  name                      = "hmtcs-${split(".", var.shutter_apps[count.index])[0]}-shutter"
+  profile_name              = azurerm_cdn_profile.main.name
+  location                  = "West US"
```

Comment:
why West US?

Reply:
CDN profile is global and in order to work with global, it needs to have WEST US in the location.